src/core/Schemas.ts

```diff
@@ -272,8 +272,10 @@ export const MoveWarshipIntentSchema = BaseIntentSchema.extend({
 
 export const QuickChatIntentSchema = BaseIntentSchema.extend({
   type: z.literal("quick_chat"),
-  recipient: z.string(),
-  quickChatKey: z.string(),
+  recipient: ID,
+  quickChatKey: z
+    .string()
+    .regex(/^(help|attack|defend|greet|misc)\.[a-zA-Z0-9_]+$/),
```

Comment:
Can you put a limit on the suffix? I didn't test this but I think something like this could work:
```suggestion
    .regex(/^(help|attack|defend|greet|misc)\.[a-zA-Z0-9_]{0,10}$/),
```